pkg/features/kube_features.go

```diff
@@ -327,6 +330,11 @@ func Enabled(f featuregate.Feature) bool {
 	return utilfeature.DefaultFeatureGate.Enabled(f)
 }
 
+// Disabled is helper for `!utilfeature.DefaultFeatureGate.Enabled()`
+func Disabled(f featuregate.Feature) bool {
+	return !Enabled(f)
+}
```

Comment:
Why just not use !Enabled(f)?

Reply:
yes, I don't see value in the extra helper